firewall/firewall_linux.go

```diff
@@ -4,13 +4,21 @@ import (
 	"errors"
 	"net"
 	"os/exec"
+	"strings"
 
 	"github.com/coreos/go-iptables/iptables"
 	"github.com/google/nftables"
 	"github.com/gravitl/netmaker/logger"
 	"github.com/vishvananda/netlink"
 )
 
+const (
+	TESTIPV4    = "1.1.1.1"
+	TESTIPV6    = "2606:4700:4700::1111"
+	IPV4NETWORk = "0.0.0.0/0"
+	IPV6NETWORk = "::/0"
+)
+
```

Comment:
Naming convention for constants in GO is mixed caps
https://go.dev/doc/effective_go#mixed-caps

Reply:
Good point.  Added one commit to change to MixedCaps for the constants.  Please help review again.  Thanks.